shared/version/api.go

```diff
@@ -411,6 +411,7 @@ var APIExtensions = []string{
 	"shared_custom_block_volumes",
 	"instance_import_conversion",
 	"instance_create_start",
+	"container_protection_start",
```

Comment:
should be `instance_protection_start` as it covers VMs as well

Reply:
Agree.

And how about `container_protection_delete`? I think it has the same problem, do we need to rename it?

Reply:
No, because that would break anyone checking for it.

Reply:
Done.